docs/infrastructure_and_maintenance/clustering/clustering_using_ddev.md

```diff
@@ -0,0 +1,186 @@
+---
+description: When you want to run locally a cluster infrastructure using DDEV.
+---
+
+# Clustering using DDEV
+
+This guide follows [Getting started: Install using DDEV](../../getting_started/install_using_ddev.md) and helps to extend the previous installation to replicate locally a production [cluster](clustering.md).
+
+Contrary to production cluster, there will be only one front app server. But the data sharing needed by a cluster of several servers can still be emulated.
+
+The `ddev config --php-version` option should set the same PHP version as the production servers.
+
+!!! caution
+
+    This is not to be used in production.
```

Comment:
I'd move this warning to the top of the page, if it refers to the whole procedure

Reply:
Done in 6c463bd4ea435441bb9c69f89724112f2461bd09